_includes/parse-server/getting-started.md

```diff
@@ -8,18 +8,20 @@ Parse Server is an open source backend that can be deployed to any infrastructur
 
 **Prerequisites**
 
-* Node 8 or newer
-* MongoDB version 3.6
-* Python 2.x (For Windows users, 2.7.1 is the required version)
-* For deployment, an infrastructure provider like Heroku or AWS
+* Node
+* MongoDB (to use Parse with MongoDB)
+* PostgreSQL (to use Parse with PostgreSQL)
+* For deployment, an infrastructure provider like Heroku, DigitalOcean or AWS
```

Comment:
It seems that this guide actually refers to a local set-up for Parse Server, so maybe we should not mention a cloud services provider as a requirement. I would also mention them in order of popularity.

Let's add as a separate paragraph:

> The following guide describes how to set up Parse Server on your personal computer for local development. If you want to be able to access Parse Server from anywhere and make your app accessible publicly, you would want to deploy Parse Server to a cloud service provider like Amazon Web Services, Google Cloud, Microsoft Azure, Heroku or DigitalOcean. These providers vary in set-up complexity, configuration efforts, pricing model and required knowledge to secure your deployment. You can find guides for how to deploy Parse Server for specific providers in other parts of this documentation.
>
> ⚠️ Before making Parse Server accessible publicly, we strongly recommend to review all of your Parse Server configuration and read our [best practice](https://docs.parseplatform.org/parse-server/guide/#best-practice) guide. Failing to properly adapt your Parse Server configuration for a publicly accessible environment may make your deployment vulnerable to malicious intrusions, data leaks and unexpected cost increases.

Could you just change the best practice link to a relative link?

Reply:
I added the link to the "Deploying Parse Server", check if is ok.